contracts/cw-cyber-airdrop/src/execute.rs

```diff
@@ -41,12 +41,16 @@ pub fn instantiate(
 
     let config = Config {
         owner: Some(owner),
+        passport: deps.api.addr_validate(&msg.passport)?,
```

Comment:
```suggestion
        passport_addr: deps.api.addr_validate(&msg.passport_addr)?,
```
Best to add `addr` suffix, makes it easier to understand in the code.